### PR TITLE
add job e2e test (E00005)

### PR DESCRIPTION
**Pull request by yulng**
add job e2e test (E00005)

### CODE REVIEW

File: test/e2e/ippool/job_pod_test.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+			err = frame.DeleteNamespace(nsName)
+			Expect(err).NotTo(HaveOccurred(), "failed to delete namespace %v", nsName)
+		})
+	})
+
+	It("one Job 2 pods allocate/release ipv4 and ipv6 addresses", Label("smoke", "E00005"), func() {
+
+		// create Job
+		GinkgoWriter.Printf("try to create Job %v/%v \n", jdName, nsName)
+		behavior := "notTerminate"
```
Comment (weizhoublue) [inline test/e2e/ippool/job_pod_test.go:43]: 三种 behavior := "notTerminate" xxx  xxxx
用 table 来写，压低 代码量

Comment (yulng) [reply test/e2e/ippool/job_pod_test.go:43]: > 三种 behavior := "notTerminate" xxx xxxx 用 table 来写，压低 代码量

已加entry

**Review by ty-dc — COMMENTED** [review]
err = frame.CheckPodListIpReady(podlist)
Expect(err).NotTo(HaveOccurred(), "failed to checkout ipv4、ipv6")

**Comment by yulng** [comment]
submit another

**Pull request closed without merging**